qemu/tests/ovs_host_vlan.py

```diff
@@ -306,5 +307,8 @@ def run(test, params, env):
                                                s_session=sessions[0],
                                                d_session=sessions[1],
                                                file_transfer_timeout=f_tmout)
+        for session in sessions:
+            if session:
+                session.close()
```

Comment:
What about move it to `finally:`, in case it fails some where?

Reply:
I am worried if case failed before creating sessions, move this to `finally` but in fact `sessions` have not yet been defined.

Reply:
What about move `sessions = []` before `try`

Reply:
Sounds good to me